sqlalchemy_ibmi/base.py

```diff
@@ -47,6 +47,7 @@
 * ``use_system_naming`` - If ``True``, the connection is set to use the System
    naming convention, otherwise it will use the SQL naming convention.
    Defaults to ``False``.
+* ``trim_char_fields`` - If ``True``, all character fields will be returned with trailing spaces truncated.
```

Comment:
```suggestion
* ``trim_char_fields`` - If ``True``, all character fields will be returned with trailing spaces truncated. Defaults to ``False``.
```

This also needs to be wrapped properly to 80 columns.